src/controllers/controlpickermenu.cpp

```diff
@@ -1479,14 +1479,23 @@ void ControlPickerMenu::addPlayerControl(const QString& control,
     }
 
     if (samplerControls) {
-        QMenu* samplerControlMenu = new QMenu(tr("Samplers"), controlMenu);
-        controlMenu->addMenu(samplerControlMenu);
+        QMenu* samplerControlMainMenu = addSubmenu(tr("Samplers"), controlMenu);
```

Comment:
A p prefix would be nice.

Reply:
Done. For all menus, the diff is big but it's done.